KerbalStuff/blueprints/mods.py

```diff
@@ -36,12 +36,15 @@ def _get_mod_game_info(mod_id):
 
 
 def _restore_game_info():
-    game_id = session.get('gameid') or 'kerbal-space-program'
-    ga = Game.query.filter(Game.id == game_id).order_by(desc(Game.id)).first()
-    if not ga:
-        abort(404)
-    set_game_info(ga)
-    return ga
+    game_id = session.get('gameid')
+    if game_id:
+        game = Game.query.filter(Game.active == True, Game.id == game_id).order_by(desc(Game.id)).first()
+    else:
+        # Default to KSP
+        game = Game.query.filter(Game.active == True, Game.short == 'kerbal-space-program').order_by(desc(Game.id)).first()
```

Comment:
What do the ID values look like in this table? Are they positive integers starting with `1`? It might make more sense to pick the lowest ID as the default rather than hard code the name. Or maybe put something in `_cfg` so the admin can choose.

Reply:
Yes, AFAIK they are generated by the DB and start at `1`.
I fear that this fallback is used fairly frequently in production, and I don't know if KSP has the id `1` there.

But thinking more about it, I'm wondering if it's even necessary to provide a default/fallback.
It's only ever used in `create_mod`, and I can easily change the template so it can handle `None` game, in which case the dropdown would have no item pre-selected and the user would have to select one.